### ISSUE TITLE
Need better error messages

### ISSUE THREAD

**Issue by rpiskule-syn** [blackducksoftware/synopsys-operator#809]
```
E0903 19:47:11.415711       1 reflector.go:126] pkg/mod/k8s.io/client-go@v11.0.0+incompatible/tools/cache/reflector.go:94: Failed to list *v1.Blackduck: v1.BlackduckList.Items: []v1.Blackduck: v1.Blackduck.Spec: v1.BlackduckSpec.Environs: []string: ReadString: expects " or n, but found {, error found in #10 byte of ...|LOADS:0",{}],"expose|..., bigger context ...|ckduck.com","USE_ALERT:0","USE_BINARY_UPLOADS:0",{}],"exposeService":"NONE","externalPostgres":{"pos|...
E0903 19:47:12.425421       1 reflector.go:126] pkg/mod/k8s.io/client-go@v11.0.0+incompatible/tools/cache/reflector.go:94: Failed to list *v1.Blackduck: v1.BlackduckList.Items: []v1.Blackduck: v1.Blackduck.Spec: v1.BlackduckSpec.Environs: []string: ReadString: expects " or n, but found {, error found in #10 byte of ...|LOADS:0",{}],"expose|..., bigger context ...|ckduck.com","USE_ALERT:0","USE_BINARY_UPLOADS:0",{}],"exposeService":"NONE","externalPostgres":{"pos|...
```
**What would you like to be added**:
I need error messages to print out what Hub is causing the error.

**Why is this needed**:
If a hub is created incorrectly, it is impossible to determine which hub is causing the problem by looking at the error message.

I propose doing something like this (psuedocode)
```
try {
   //run all your operator stuff here
} catch (exception e) {
   log.error("Namespace" + namespace + " caused an issue"
   throw e
}
```

**Comment by msenmurugan** [comment]
relates to #905

**Comment by msenmurugan** [comment]
Will fix as part of #905
